module.c

```diff
@@ -129,6 +136,42 @@ static void object_dealloc(ObjectData *self) {
 	PyObject_GC_Del(self);
 }
 
+// _quickjs.Object.get
+//
+// Gets a Javascript property of the object.
+static PyObject *object_get(ObjectData *self, PyObject *args) {
+	const char *name;
+	if (!PyArg_ParseTuple(args, "s", &name)) {
+		return NULL;
+	}
+	JSValue value = JS_GetPropertyStr(self->context->context, self->object, name);
+	return quickjs_to_python(self->context, value);
+}
+
+// _quickjs.Object.set
+//
+// Sets a Javascript property to the object.
+static PyObject *object_set(ObjectData *self, PyObject *args) {
+	const char *name;
+	PyObject *item;
+	if (!PyArg_ParseTuple(args, "sO", &name, &item)) {
+		return NULL;
+	}
+	int ret = 0;
+	if (python_to_quickjs_possible(self->context, item)) {
+		ret = JS_SetPropertyStr(self->context->context, self->object, name,
+		                        python_to_quickjs(self->context, item));
+		if (ret != 1) {
+			PyErr_SetString(PyExc_TypeError, "Failed setting the variable.");
+		}
+	}
+	if (ret == 1) {
+		Py_RETURN_NONE;
+	} else {
+		return NULL;
```

Comment:
Need an error message if `python_to_quickjs_possible` fails. 

The unit tests should also cover the failure cases.

Reply:
Actually, `python_to_quickjs_possible` *does* set an error message when failing, so that's fine. Still agreeing on the need for unit tests.